_data/people.yml

```diff
@@ -79,6 +79,11 @@
 
 # Graduate Students
 # [in order of first appearance in the lab]
+ - name: Tanmay Desai
+   role: MS
```

Comment:
We don't have this role defined [here](https://github.com/HIRO-group/HIRO-group.github.io/blob/f80f8dc817531c22637cf22d124079f0484b6a98/people.html). This is why your pic isn't showing up.

Reply:
Uncomment 46-48 in the linked file and update your role to `grad`.